packages/daimo-api/src/router.ts

```diff
@@ -113,9 +113,31 @@ export function createRouter(
           throw new Error(`Invalid Daimo app link: ${url}`);
         }
 
+        async function getEAccountFromLinkString(
```

Comment:
what about

`getEAccountFromStr(eAccountStr)`

and then for display we use `getEAccountDisplayName` or similar, call it `displayName` etc

Reply:
done